Library/ViewModels/ProjectPamphletViewModelTests.swift

```diff
@@ -686,7 +862,7 @@ final class ProjectPamphletViewModelTests: TestCase {
       self.goToRewardsProject.assertDidNotEmitValue()
       self.goToRewardsRefTag.assertDidNotEmitValue()
 
-      self.vm.inputs.backThisProjectTapped()
+      self.vm.inputs.pledgeCTAButtonTapped(.pledge)
```

Comment:
See, I think this is where a better naming convention would help us a little `Tapped(with: .pledge)`..What do you think?

Reply:
Oh right. I changed in one place but missed that one. Will change it